webapp/src/components/Asset/Asset.js

```diff
@@ -33,8 +33,8 @@ export default class Asset extends React.PureComponent {
   }
 
   componentWillMount() {
-    const { value, isLoading, onFetchAsset } = this.props
-    if (value || isLoading) {
+    const { isLoading, onFetchAsset } = this.props
```

Comment:
Could you remind me why are we changing this? it's a fairly delicate piece of code :P

cc: @cazala

Reply:
In order to re-fetch the asset when showing the detail